src/caselawclient/models/identifiers/fclid.py

```diff
@@ -0,0 +1,48 @@
+import re
+from typing import TYPE_CHECKING
+
+from sqids import Sqids
+
+from . import Identifier, IdentifierSchema
+
+if TYPE_CHECKING:
+    from caselawclient.Client import MarklogicApiClient
+
+
+VALID_FCLID_PATTERN = re.compile(r"^[bcdfghjkmnpqrstvwxyz23456789]{4,}$")
+
+FCLID_MINIMUM_LENGTH = 8
+FCLID_ALPHABET = "bcdfghjkmnpqrstvwxyz23456789"
```

Comment:
The repeated string irks me... `r"^[" + alphabet + r"]{4,}$"` isn't cleaner, though, nor is `{alphabet}{{4,}}`...